src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Subclass of Task, for events happening on certain time
+ */
+public class Event extends Task {
+    protected String at;
```

Comment:
Consider using a more define name for clearer understanding.

Reply:
Updated to "date". Thanks for the suggestion!